app/main.py

```diff
@@ -1,6 +1,22 @@
+import requests
+import os
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    weather_site = "https://api.weatherapi.com/v1/current.json?key="
```

Comment:
And better have it outside function as constant
`URL = 'https://api.weatherapi.com/v1/current.json"`